src/lsp/cobol_lsp/lsp_completion.ml

```diff
@@ -173,25 +126,25 @@ let expected_tokens base_env =
       | None -> snd (Srcloc.as_lexloc Srcloc.dummy)
       | Some Menhir.Element (_, _, _, pos) -> pos in
     if debug then (Lsp_io.log_debug "In State: %a" pp_env env; let tok = Expect.acceptable_terminals_in ~env in if List.length tok > 0 then Lsp_io.log_debug "Gained %d entries [%a]" (List.length tok) Fmt.(list ~sep:(any ";") Expect.pp_completion_entry) tok);
-    let productions = Expect.reducable_productions_in ~env in
```

Comment:
All that `expected_tokens` is a bit messy. But we can leave it as is and anybody wanting to understand will refactor a bit first (that's typically what happens in OCaml: refactoring helps understanding).